README.md

```diff
@@ -120,6 +123,30 @@ need to be provided in order for this action to successfully run using one of th
 | ------------------------- | -------- | ----------- |
 | `imageToDeploy`           | Yes (for this scenario) | The name of the image that has already been pushed to a registry and will be deployed to the Container App by this action. If this image is found in an ACR instance that requires authentication to pull, the `acrName` argument, or the `acrUsername` and `acrPassword` arguments, can be provided to authenticate requests to the ACR instance. |
 
+### Arguments required for using a YAML configuration file
+
+| Argument name             | Required | Description |
+| ------------------------- | -------- | ----------- |
+| `yamlConfigPath`          | Yes (for this scenario) | Full path (on the executing GitHub runner) to the YAML file detailing the configuration of the Container App. |
+
+#### Important note on the YAML configuration file

```

Comment:
I don't know if it is something we want to call out, or just hope that it will get fixed soon, but I do want to note that currently YAML config does not support managed identity auth.

Issue: https://github.com/microsoft/azure-container-apps/issues/524
Work item: https://msazure.visualstudio.com/One/_workitems/edit/16454602/

Reply:
Good callout 👍 I'll make sure that this is noted.